src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -65,7 +65,7 @@ class WorkflowView extends Component {
 
   getModifiedConfig() {
     const { config, modifiedAttributes } = this.state
-    return _.merge(config, modifiedAttributes)
+    return _.assign(config, modifiedAttributes)
```

Comment:
I don't think this is right. `modifiedAttributes` is a nested object that contains deltas, and needs to be deeply merged with the original config.